tools/deb/istio-iptables.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
@rshriram we should unify eventually we agree but for now we want to make sure we don't leave a VM in unreachable state, and we need to iterate on the VM iptables faster/make more changes than the regular ones. So the unification will be post 0.2